rever.xsh

```diff
@@ -0,0 +1,33 @@
+$PROJECT = $GITHUB_ORG = $GITHUB_REPO = 'xonsh'
+$WEBSITE_URL = 'http://xon.sh'
+$ACTIVITIES = ['version_bump', 'changelog', 'pytest',
+               'sphinx', 'tag', 'pypi', 'conda_forge', 'ghpages', 'ghrelease'
+               ]
+
+$VERSION_BUMP_PATTERNS = [
+    ('.appveyor.yml', 'version:.*', 'version: $VERSION.{build}'),
+    ('xonsh/__init__.py', '__version__\s*=.*', "__version__ = '$VERSION'"),
+    ]
+$CHANGELOG_FILENAME = 'CHANGELOG.rst'
+$CHANGELOG_TEMPLATE = 'TEMPLATE.rst'
+
+#$PYTEST_COMMAND = "rm -r .cache/ __pycache__/ */__pycache__ */*/__pycache__ && pytest"
```

Comment:
Non-blocking, but should this just be removed?

Reply:
Doh! Yes!

Reply:
Done!